guides/ansible_engine/5-role/README.md

```diff
@@ -0,0 +1,221 @@
+# Exercise 1.5 - Roles: Making your playbooks reusable
+
+While it is possible to write a playbook in one file as we've done throughout this workshop, eventually you’ll want to reuse files and start to organize things.
+
+Ansible Roles is the way we do this.  When you create a role, you deconstruct your playbook into parts and those parts sit in a directory structure.  "Wha??  You mean that seemingly useless [best practice](http://docs.ansible.com/ansible/playbooks_best_practices.html) you mentioned in exercise 1.2?".  Yep, that one.
+
+For this exercise, you are going to take the playbook you just wrote and refactor it into a role.  In addition, you'll learn to use Ansible Galaxy.
+
+Let's begin with seeing how your apache-basic-playbook will break down into a role.
+
+![Role directory structure](roledir_1.png)
```

Comment:
I'd rather this be represented as plain text block than a screenshot. It's not necessary and it's easier to edit if needed.

Reply:
@liquidat Agreed with @tima